core/auto_process/managers/sickbeard.py

```diff
@@ -303,9 +300,13 @@ def detect_fork(self):
         return fork
 
     def _init_fork(self):
-        from .pymedusa import PyMedusa
+        # These need to be imported here, to prevent a circular import.
+        from .pymedusa import PyMedusa, PyMedusaApiV1, PyMedusaApiV2
+
         mapped_forks = {
-            'Medusa': PyMedusa
+            'Medusa': PyMedusa,
+            'Medusa-api': PyMedusaApiV1,
+            'Medusa-apiv2': PyMedusaApiV2
```

Comment:
you could add a line here for the default sickbeard fork. And use the class `SickBeard` for the same file. But i'm not sure for which "forks" you would want to use that. So I haven't added it

Reply:
Thanks, I'll take a look and see what forks to add here.

As to wiki, absolutely. Probably need to update https://github.com/clinton-hall/nzbToMedia/wiki/sickbeard-forks and create a new Medusa Page